server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -1286,6 +1288,7 @@ protected Node(
                 b.bind(Tracer.class).toInstance(tracer);
                 b.bind(SearchRequestStats.class).toInstance(searchRequestStats);
                 b.bind(SearchRequestSlowLog.class).toInstance(searchRequestSlowLog);
+                b.bind(SearchRequestCoordinatorTrace.class).toInstance(searchRequestCoordinatorTrace);
```

Comment:
I don't think it is need, we are not supposed to inject listener to be used out of the context